modules/ROOT/pages/organization.adoc

```diff
@@ -35,44 +31,44 @@ An Anypoint Platform user who has been granted organization administrator privil
 * Access analytics for the APIs in your organization
 * Create business groups to delegate management of the resources and define the scopes of roles and permissions. A business group owner can configure additional properties at the business group level.
 
+When the organization owner creates a business group, an owner must be assigned to it. This assigned user holds an *Organization Administrators* role within that business group by default.
```

Comment:
but since the creator is automatically given this role, are we saying a 2nd owner must be assgined? Or "owner" has some meaning outside of Organization Administrators role?

Reply:
Owner is not a role. Every business group has to have an owner and every owner is automatically assigned the role of Organization Administrator.